MiniKeePass/Lock Screen/LockScreenManager.m

```diff
@@ -26,13 +32,14 @@
 #import "PasswordUtils.h"
 
 @interface LockScreenManager () <PinViewControllerDelegate>
-@property (nonatomic, strong) LockViewController *lockViewController;
```

Comment:
It looks like your implementation no longer uses the LockViewController class.  If so, that class should be deleted and the deletion included in the pull request.